encrypted-media/scripts/playback-temporary-waitingforkey.js

```diff
@@ -16,52 +20,30 @@ function runTest(config) {
             _mediaKeySessions = [ ],
             _mediaSource;
 
-        function onMessage(event) {
-            assert_any( assert_equals, event.target, _mediaKeySessions );
-            assert_true( event instanceof window.MediaKeyMessageEvent );
-            assert_equals( event.type, 'message');
-
-            assert_any( assert_equals,
-                        event.messageType,
-                        [ 'license-request', 'individualization-request' ] );
+        function onFailure( error ) {
+            forceTestFailureFromPromise(test, error);
+        }
 
+        function onMessage(event) {
+        
             config.messagehandler( config.keysystem, event.messageType, event.message )
             .then( function( response ) {
-
                 event.target.update( response )
-                .catch(function(error) {
-                    forceTestFailureFromPromise(test, error);
-                });
+                .catch(onFailure);
```

Comment:
nite: I find this (lack of) indentation confusing. Elsewhere where we don't have such indentation, the closing parenthesis is immediately before the ')'. I'm not sure that's better. Would just moving this to the line above be better?

ditto on line 42.